estate/models/real_estate_property.py

```diff
@@ -0,0 +1,29 @@
+from odoo import fields, models
+
+class TestModel(models.Model):
+    _name = "test_model_real_estate"
+    _description = "Test Model of Real estate"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
```

Comment:
```suggestion
    is_garage = fields.Boolean()
    is_garden = fields.Boolean()
```
This way of defining fields would be more effective while using this in codes as easily identified as a boolean field.